### PR TITLE
Update Pants Version

### PR DESCRIPTION
**Pull request by christophermaier**
Update Pants to 2.13.0
Changelog: https://pypi.org/project/pantsbuild.pants/2.13.0/

---

*This change was executed automatically with [Shepherd](https://github.com/NerdWalletOSS/shepherd).* 💚🤖

### CODE REVIEW

File: 3rdparty/python/constraints.txt [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-protobuf==3.20.1
-pulumi==3.35.2
-pulumi-aws==5.9.2
+pip==22.2.2
+protobuf==4.21.5
+pulumi==3.39.3
+pulumi-aws==5.13.0
 pulumi-policy==1.4.0
-PyYAML==5.4.1
+PyYAML==6.0
```
Comment (twunderlich-grapl) [inline 3rdparty/python/constraints.txt:13]: Breaking change: yaml.load now requires the Loader argument

File: 3rdparty/python/constraints.txt [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+attrs==22.1.0
 dill==0.3.5.1
 grpcio==1.47.0
 parver==0.3.1
-pip==22.1.2
-protobuf==3.20.1
-pulumi==3.35.2
-pulumi-aws==5.9.2
+pip==22.2.2
+protobuf==4.21.5
```
Comment (twunderlich-grapl) [inline 3rdparty/python/constraints.txt:9]: potentially breaking changes,

**Review by twunderlich-grapl — APPROVED** [review]
Approving but with a small warning that some of the upgrades may have breaking changes that could affect this repo.

**Comment by christophermaier** [comment]
> Approving but with a small warning that some of the upgrades may have breaking changes that could affect this repo.

These are all dependencies that are brought in through Pulumi; they're not used directly by anything in this repository, so it should be fine.

**Pull request merged**